static/app/views/releases/thresholdsList/index.tsx

```diff
@@ -0,0 +1,33 @@
+import {RouteComponentProps} from 'react-router';
+
+import {Organization, PageFilters, Project} from 'sentry/types';
+import withOrganization from 'sentry/utils/withOrganization';
+import withPageFilters from 'sentry/utils/withPageFilters';
+import withProjects from 'sentry/utils/withProjects';
+
+import Header from '../components/header';
+
+type RouteParams = {
+  orgId: string;
+};
+
+type Props = RouteComponentProps<RouteParams, {}> & {
+  organization: Organization;
+  projects: Project[];
+  selection: PageFilters;
+};
+
+function ReleaseThresholdList({router, organization}: Props) {
+  const hasV2ReleaseUIEnabled = organization.features.includes('release-ui-v2');
+  if (!hasV2ReleaseUIEnabled) {
+    router.replace('/releases/');
+  }
+
+  return (
+    <div>
+      <Header router={router} hasV2ReleaseUIEnabled />
+    </div>
+  );
+}
+
+export default withProjects(withOrganization(withPageFilters(ReleaseThresholdList)));
```

Comment:
we're trying to use hooks on new components instead of HoC. i think these should all exist like `usePageFilters`

Reply:
Oh awesome.
Love that direction.

will update